app/models/user.rb

```diff
@@ -85,7 +85,7 @@ def self.new_token
   end
 
   def admin
-    return true if self.email == "lhsdvm@aol.com"  || self.email == "testuser@berkeley.edu"
+    return true if self.email == "lhsdvm@aol.com"  || self.email == "testuser@berkeley.edu"  || self.email == "xshuyin@berkeley.edu"
```

Comment:
Did you mean to have this here or should this be commented out? In the future might want to make like a development version or something maybe?

Reply:
got it!

Reply:
oh never mind it's already commented on by eric